internal/impl/sql/conn_fields.go

```diff
@@ -78,6 +78,12 @@ CREATE TABLE IF NOT EXISTS some_table (
 			Optional().
 			Advanced().
 			Version("1.0.0"),
+		service.NewBoolField("init_verify_conn").
+			Description("Whether to verify the database connection on startup by performing a simple ping. Disabling this allows for certain components to lazily connect.").
```

Comment:
```suggestion
			Description("Whether to verify the database connection on startup by performing a simple ping, by default this is disabled.").
```

nit: Think that this is clearer that the default is set to false?

Reply:
Yeah that’s fair. You think it’s worth mentioning lazy loading?

Reply:
not sure - if just me or I don't know exactly what is meant lazy loading?

Reply:
just think if it is worded so clearer to explain what the default behaviour is

Reply:
Added your suggestion 👍